substrate/frame/sudo/src/benchmarking.rs

```diff
@@ -21,14 +21,25 @@ use super::*;
 use crate::Pallet;
 use alloc::{boxed::Box, vec};
 use frame_benchmarking::v2::*;
+use frame_support::dispatch::DispatchInfo;
 use frame_system::RawOrigin;
+use sp_runtime::traits::{
+	AsAuthorizedOrigin, AsSystemOriginSigner, DispatchTransaction, Dispatchable,
+};
 
 fn assert_last_event<T: Config>(generic_event: crate::Event<T>) {
 	let re: <T as Config>::RuntimeEvent = generic_event.into();
 	frame_system::Pallet::<T>::assert_last_event(re.into());
 }
 
-#[benchmarks(where <T as Config>::RuntimeCall: From<frame_system::Call<T>>)]
+#[benchmarks(where
+	T: Send + Sync,
+	<T as Config>::RuntimeCall: From<frame_system::Call<T>>,
+	<T as frame_system::Config>::RuntimeCall: Dispatchable<Info = DispatchInfo>,
+	<<T as frame_system::Config>::RuntimeCall as Dispatchable>::PostInfo: From<()>,
+	<<T as frame_system::Config>::RuntimeCall as Dispatchable>::RuntimeOrigin:
+		AsSystemOriginSigner<T::AccountId> + AsAuthorizedOrigin + Clone,
```

Comment:
They are all needed because of the trait bounds on the `TransactionExtension` impl on `CheckOnlySudoAccount` and/or the benchmarking code in this file, specifically the extension benchmark

```
#[benchmark]
fn check_only_sudo_account() { ... }
```